sql/ha_partition.cc

```diff
@@ -8647,6 +8647,11 @@ int ha_partition::change_partitions_to_open(List<String> *partition_names)
      HA_EXTRA_NO_READCHECK=5                 No readcheck on update
      HA_EXTRA_READCHECK=6                    Use readcheck (def)
 
+  HA_EXTRA_REMEMBER_POS:
+  HA_EXTRA_RESTORE_POS:
+    System versioning uses this for MyISAM and Aria tables. This is needed for
+    cases where Write in a middle of Search and Delete 'corrupts' saved rowid.
```

Comment:
I don't like comment here and in the description. This legend does not help to locate specific code and is partially true (no corruption, not Write but `ha_update()`). In commit description there is more discrepancy from the truth, so please be more concise and precise.